### ISSUE TITLE
Pick with contacts Image - Feature Request

### ISSUE THREAD

**Issue by anandsubbu007** [DRSchlaubi/contact_picker#7]
Can i get Contacts picture data when picking from native contact book ?

**Comment by DRSchlaubi** [comment]
I don't now if you can actually send an image through a message channel

**Comment by maheshmnj** [comment]
> I don't now if you can actually send an image through a method channel

I think we can send the image as array of bytes and then convert it back to an image using [Image.memory](https://api.flutter.dev/flutter/widgets/Image/Image.memory.html) 
here's a discussion on it ,I haven't tried it out myself by I am assuming this would work
``` 
https://stackoverflow.com/questions/51516310/sending-bitmap-to-flutter-from-android-platform
```

**Comment by DRSchlaubi** [comment]
I'll See if I can get that to work

**Comment by DRSchlaubi** [comment]
Fixed in 3.0.0
